src/features/user/Account/components/MembershipCta.tsx

```diff
@@ -0,0 +1,41 @@
+import MuiButton from '../../../common/InputTypes/MuiButton';
+import PlanetLogo from '../../../../../public/assets/images/PlanetLogo';
+import styles from '../AccountHistory.module.scss';
+import { useTranslation, Trans } from 'next-i18next';
+import { getDonationUrl } from '../../../../utils/getDonationUrl';
+import { useUserProps } from '../../../common/Layout/UserPropsContext';
+
+interface Props {
+  placement: 'top' | 'right';
+}
+
+const MembershipCta = ({ placement }: Props) => {
+  const { t } = useTranslation('me');
+  const { token } = useUserProps();
+
+  return (
+    <a
+      className={`${styles.membershipCta} ${
+        placement === 'top'
+          ? styles.membershipCtaTop
+          : styles.membershipCtaRight
+      }`}
+      href={getDonationUrl('proj_LOxkf5GYI054Fi0HcEUF3dKu', token)}
```

Comment:
Is it ok for hardcode the project id `proj_LOxkf5GYI054Fi0HcEUF3dKu` here - will it be the same for the production and staging system and not getting changed that often?

Reply:
This is tentative, on the assumption that the project does not change (as the CTA drives contributions to the membership project)

@sagararyal your input here?

Reply:
it can be hardcoded as long as it is id from prod.